src/main/java/uk/co/ramp/covid/simulation/population/Pensioner.java

```diff
@@ -1,5 +1,25 @@
 package uk.co.ramp.covid.simulation.population;
 
+import uk.co.ramp.covid.simulation.CovidParameters;
+import uk.co.ramp.covid.simulation.DailyStats;
+
 public class Pensioner extends Person {
 
+    @Override
+    public void reportInfection(DailyStats s) {
+        s.incInfectionsPensioner();
+    }
+
+    @Override
+    public void reportDeath(DailyStats s) {
+        s.incDeathsPensioner();
+    }
+
+    @Override
+    public void allocateCommunalPlace(Population p) {}
```

Comment:
Do pensioners need to be allocated to communal places?

Reply:
Not currently. This is used to assign people workplaces/schools/nurseries - Pensioners currently only go to restaurants/shopping (which are handled separately). Probably useful to have this hook for future extensions to to keep the API uniform.